lib/prismaAPL.ts

```diff
@@ -0,0 +1,41 @@
+import { APL, AuthData } from "../sdk/types/apl";
```

Comment:
Example of the custom APL. Prisma will not be merged with the template and will be a separate example.